packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -186,6 +186,13 @@ export default class Sayt extends LitElement {
    */
   render() {
     return html`
+      <style>
+        ${ this.visible ? `
+          sfx-sayt {
+            display: block;
+          }
+        ` : ''}
+      </style>
```

Comment:
Note that this does create empty `<style>` tags when invisible.

Reply:
If there happens to be multiple `sfx-sayt` components on the page, this is not going to work. What would be better is to define two static styles, one for visible and one for hidden:

```css
sfx-sayt {
  display: none;
}

sfx-sayt[visible] {
  display: block;
}
```

Or the other way:
```css
sfx-sayt {
  display: block;
}

sfx-sayt[hidden] {
  display: none;
}
```

The browser should already have styles for `[hidden]`, so I wonder if simply defining `sfx-sayt` to be `display: block` is enough.

Reply:
Good catch! I didn't notice it, despite it being apparent in a Storybook story. Will change.